### ISSUE TITLE
Retain Table Name Casing

### ISSUE THREAD

**Issue by TheBuzzSaw** [pvanek/sqliteman#10]
The table list converts each name to all lowercase. This makes the table list harder to read if the convention used was camel casing. Is there a reason they are all forced to lowercase?

**Comment by Austentacious** [comment]
I'm running into this also.  I have some descriptive table names and it always displays them lowercase and they are much harder to read.  I did notice that if I do a DESCRIBE on the table, it will show it as being mixed case.  Hate to hear that an alter can cause the casing to change.  Is this something that might get fixed?

**Comment by TheBuzzSaw** [comment]
Considering I opened this a year ago, maybe I just need to fork it and attempt the fix myself.

**Comment by rparkins999** [comment]
This (as well as many other bugs) is fixed in rparkins999/sqliteman.

**Comment by sbhoek** [comment]
Hi rparkins999, Did you get a volunteer already to build it for Windows? I have a Windows system with Visual Studio 2008. I wish I knew how to build it, but I have no real clue. Use CMake or so???

**Comment by TheBuzzSaw** [comment]
I dunno that I'll ever address this myself. I've long since moved onto other tools.

**Comment by rparkins999** [comment]
No, I didn't get a volunteer to build on Windows. If you can get a command window, cmake and the make should build it. If you try it and it doesn't work, post the output from both the cmake and the make and I'll try and see if I can fix it. If it does work, please do a pull request with the .exe file.

**Comment by rparkins999** [comment]
If you clone the rparkins999/sqliteman repository, navigate to Sqliteman/sqliteman/doc/en/, and open index.html with your favourite browser. you will get into sqliteman's help system. Appendix A tells you how to build and install it and what libraries are required. I think you need to download sqlite.c from the sqlite website and make a small edit to it: this is described somewhere in the release notes in githib.

**Comment by sbhoek** [comment]
Thank you for the instructions. I don't have time to work on this without interruption, but I intend to do it in between other tasks. I've started by installing CMake on Windows and on my Xubuntu instance, which I run under VirtualBox. I'll keep you updated ...
 

Gesendet: Dienstag, 28. August 2018 um 19:21 Uhr
Von: rparkins999 <notifications@github.com>
An: pvanek/sqliteman <sqliteman@noreply.github.com>
Cc: Steven <dobedani@gmx.net>, Comment <comment@noreply.github.com>
Betreff: Re: [pvanek/sqliteman] Retain Table Name Casing (#10)


... actually it's sqlite3.c, and the edited version is already included in the repository. You only have to edit if you replace sqlite3.c with a more recent version from the sqlite website. Instructions for doing this are in the file README - updating version.
